client/sites-dashboard/components/sites-table.tsx

```diff
@@ -137,6 +137,8 @@ export function SitesTable( { className, sites, isLoading = false }: SitesTableP
 		};
 	}, [ masterbarHeight ] );
 
+	const isStatusDeleted = sites.some( ( site ) => ! site.is_deleted );
```

Comment:
This probably deserves it `useMemo` wrapping so that we can avoid calculating this on every render. It's ok for smaller samples but once there are a lot of sites it becomes a bottleneck.

Reply:
It's been removed after Javier's feedback. We will keep the table columns the same, so we don't need this.